airflow/configuration.py

```diff
@@ -211,6 +211,13 @@ class AirflowConfigParser(ConfigParser):
                 3.0,
             ),
         },
+        'elasticsearch': {
```

Comment:
We should probably add this to `UPDATING.md`

Reply:
Whops, we never added a similar message for log_filename_template.

Reply:
Done (for both) in 88ee4dfe